test/e2e/benchmark/benchmark.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/celestiaorg/celestia-app/v3/pkg/appconsts"
 	"github.com/celestiaorg/celestia-app/v3/test/e2e/testnet"
 	"github.com/celestiaorg/celestia-app/v3/test/util/testnode"
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+const (
+	timeFormat = "20060102_150405"
 )
```

Comment:
[uber nit] a const block isn't necessary if there's only one constant

```suggestion
const timeFormat = "20060102_150405"
```